ydb/library/actors/wilson/wilson_uploader.h

```diff
@@ -30,6 +30,10 @@ namespace NWilson {
         TString ServiceName;
         std::unique_ptr<IGrpcSigner> GrpcSigner;
 
+        ui32 MaxSpansPerSecond = 10;
```

Comment:
Not sure whether these limits should be in this file or in kikimr_services_initializers.cpp

Reply:
I'd suggest no default limits when no respective configuration section is filled in, especially in OpenSource.

Reply:
Changed default limits to infinity effectively